loleaflet/src/control/Permission.js

```diff
@@ -81,8 +81,12 @@ L.Map.include({
 			this.fire('editorgotfocus');
 			// In the iOS/android app, just clicking the mobile-edit-button is
 			// not reason enough to pop up the on-screen keyboard.
-			if (!(window.ThisIsTheiOSApp || window.ThisIsTheAndroidApp))
-				this.focus();
+			if (!(window.ThisIsTheiOSApp || window.ThisIsTheAndroidApp)) {
+				if (this._docLayer._docType === 'spreadsheet') {
```

Comment:
Don't we want to take focus for all document types? F2 is meaningful only for Calc, but the focus is for all.

Also, I think there is .isCalc() helper function somewhere. Can you check if we can use it here please? Comparing with strings is error prone, so best to minimize/avoid them.